qcschema/dev/basis.py

```diff
@@ -21,25 +25,18 @@
             "description": "Brief description of the basis set",
             "type": "string"
         },
-        "element_basis": {
-            "description": "Per-element basis data",
+        "basis_data": {
+            "description": "Basis data shared among all atoms/centers",
```

Comment:
`for all atoms/centers`, I think, b/c want this to be a full spec for a given Mol.

In Psi, got into a nice tangle of a BasisSet owns a Mol and a Mol owns a hash repr of a BasisSet. Plan to keep these definitely decoupled, even though `len(basis_atom_map)` == `len(symbols)`?

Reply:
Ok I think you are asking for just a change in wording the description?

I agree in keeping the basis set and molecule decoupled. And certainly no circular references

Reply:
Right, just a wording change to emphasize that BasisSet should cover whole molecule.